src/components/ItemInformation.tsx

```diff
@@ -1,52 +1,52 @@
 import React from 'react';
-import { restaurant } from '../utils/interfaces';
+import { Restaurant } from '../utils/interfaces';
 import '../styles/ItemInformation.css';
 import { CATEGORY_IMAGE_PATH } from '../utils/constants';
 
 interface Props {
-  restaurant: restaurant;
+  restaurant: Restaurant | null;
   closeEvent: () => void;
 }
 
-class ItemInformation extends React.Component<Props> {
-  render(): React.ReactNode {
-    const { category, name, description, link, distance } = this.props.restaurant;
-    return (
+function ItemInformation({ restaurant, closeEvent }: Props) {
+  return (
+    restaurant && (
```

Comment:
조건부 렌더링을 이렇게 사용하는 것도 좋지만

```suggestion
if (!restaurant) return null;

return (
  <div className="modal-container">
```

이런 식으로 명시적으로 처리해줘도 좋을거 같네요~